index.bs

```diff
@@ -1592,6 +1592,8 @@ The <dfn constructor for="XRWebGLLayer">XRWebGLLayer(|session|, |context|, |laye
 
 </div>
 
+Note: When a {{XRWebGLLayerInit}}'s {{XRWebGLLayerInit/useDefaultFramebuffer}} is set to <code>true</code> all other values on the {{XRWebGLLayerInit}} object are ignored, since the {{WebGLRenderingContext}}'s default framebuffer was already allocated using the context's [=actual context parameters=] and cannot be overridden.
```

Comment:
> [=actual context parameters=]

Is this defined somewhere?

Reply:
Yes. When the .bs is built it links here: https://www.khronos.org/registry/webgl/specs/latest/1.0/#actual-context-parameters